server/container/cli-integration/src/test/resources/mailetcontainer.xml

```diff
@@ -60,7 +60,16 @@
                 <processor>local-address-error</processor>
                 <notice>550 - Requested action not taken: no such user here</notice>
             </mailet>
-            <mailet match="SMTPAuthSuccessful" class="RemoteDelivery">
+            <mailet match="SMTPAuthSuccessful" class="SetMailAttribute">
+                <RelayAllowed>true</RelayAllowed>
+            </mailet>
+            <mailet match="SentByMailet" class="SetMailAttribute">
+                <RelayAllowed>true</RelayAllowed>
+            </mailet>
+            <mailet match="org.apache.james.jmap.mailet.SentByJmap" class="SetMailAttribute">
```

Comment:
I don't see the issue it's a conf file, if you don't want jmap you just remove this line that all. If you add jmap you need the config for it. You can't have bread and butter